internal/services/service_test.go

```diff
@@ -110,7 +111,24 @@ func (s *ServiceTestSuite) TestServiceReady() {
 
 	// shutdown
 	cancel()
-	s.Nil(<-result, "service exited for the wrong reason: %v", err)
+	err = <-result
+	s.ErrorIs(err, context.Canceled, "service exited for the wrong reason: %v", err)
+}
+
+// Regression test to catch a panic if the executable is not found in $PATH
```

Comment:
Minor of minors, but: we don't need to document this as a "Regression test".
Just `test to catch a panic if the executable is not found in $PATH` is fine.